modules/ignition/resources/files/registries.conf

```diff
@@ -0,0 +1,25 @@
+# This is a system-wide configuration file used to
+# keep track of registries for various container backends.
+# It adheres to TOML format and does not support recursive
+# lists of registries.
+
+# The default location for this configuration file is /etc/containers/registries.conf.
+
+# The only valid categories are: 'registries.search', 'registries.insecure',
+# and 'registries.block'.
+
+[registries.search]
+registries = ['docker.io', 'registry.access.redhat.com']
+
+# If you need to access insecure registries, add the registry's fully-qualified name.
+# An insecure registry is one that does not have a valid SSL certificate or only does HTTP.
+[registries.insecure]
+registries = []
+
+
+# If you need to block pull access from a registry, uncomment the section below
+# and add the registries fully-qualified name.
+#
+# Docker only
+[registries.block]
+registries = []
```

Comment:
[The docs][1] are not super clear, but it seems like we could accomplish this same result by leaving this block (and `registries.insecure`) off entirely.  If that's the case, I'd rather have the file content be just:

```toml
# Configure container registries.  Docs in containers-registries.conf(5).

[registries.search]
registries = ['registry.access.redhat.com', 'docker.io']
```

or similar.

[1]: https://github.com/containers/image/blob/4d1b61cb32c6aa82a0ad2bad43350a9f5c3bc0a2/docs/containers-registries.conf.5.md

Reply:
done

Reply:
+1

Reply:
@wking I tested this out and it does appear that you can remove these sections completely.